src/components/ChatList/elements/ChannelImage.tsx

```diff
@@ -84,7 +90,7 @@ const Small: React.FC<ChannelImageBadgeProps> = ({type}) => {
     );
   }
 
-  if (type === 'GROUP') {
+  if (type === CHANNEL_GROUP) {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The condition check for the type of channel has been updated to use the constant `CHANNEL_GROUP` instead of the hard-coded string 'GROUP'. This is a good practice as it reduces the risk of typos and makes the code easier to maintain.
```diff
- if (type === 'GROUP') {
+ if (type === CHANNEL_GROUP) {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->